src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsAmazonObject.java

```diff
@@ -1,9 +1,30 @@
 package com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase;
 
+import com.wikia.webdriver.common.contentpatterns.AdsContent;
 import org.junit.Assert;
 import org.openqa.selenium.WebDriver;
 
 public class AdsAmazonObject extends AdsBaseObject {
+  private static final String QS_TURN_ON_A9 = "InstantGlobals.wgAdDriverA9BidderCountries=[XX]";
```

Comment:
I have mixed fillings about putting those consts in this class. Shouldn't page object represent specific page/DOM state?

Reply:
I agree, it should... but it isn't :) For us `Page` object is more like "ad product" from test perspective and it keeps everything what you need to test specific (in this case for example run debug mode for A9, verify if there is bid in slot etc.) functions. It should be thinner as you said, but in that case we need typical ad-product object with functions specific for every ad-product (run FV video, unmute VUAP video) but I think that would be too complicated and this way is simpler.